circle.yml

```diff
@@ -10,6 +12,7 @@ general:
 test:
   post:
     - cat ./coverage/lcov.info | ./node_modules/.bin/coveralls --verbose
+    - if [ $CIRCLE_BRANCH = 'master' ]; then npm run update; fi
```

Comment:
This shouldn't be in the `test: post` section, that test is exactly what is done by Circle in the `deployment` section  :)